package.json

```diff
@@ -14,5 +14,8 @@
     "test": "npm run test --workspaces --if-present",
     "integration": "npm run integration --workspaces --if-present",
     "coverage": "npm run coverage --workspaces --if-present"
+  },
+  "dependencies": {
+    "typescript": "^5.4.2"
```

Comment:
We shouldn't need to add this to the root since each workspace is managing their own deps. Did you run into an issue without this?

Reply:
reverted, I didn't see ts compiler is specified for each package